### ISSUE TITLE
Conditional newsletter sign up at the end of articles

### ISSUE THREAD

**Issue by DriftingSands** [headwirecom/helix-sportsmagazine#264]
Most articles have a newsletter signup box at the bottom of the page.
What the box signs you up for is based on the article.

![image](https://github.com/headwirecom/helix-sportsmagazine/assets/68734204/0536c676-da06-436f-9e56-ac269edf755e)
![image](https://github.com/headwirecom/helix-sportsmagazine/assets/68734204/f15bbf85-adda-4801-8fc3-8d08e38f534c)

**Comment by hsaginor** [comment]
@DriftingSands Can you provide some URLs?
@icaraps Should these be imported as blocks?

**Comment by icaraps** [comment]
@hsaginor yes block would be easier to implement given that there is some logic involved with the form submission probably.
